src/Http/Controllers/Configuration/UserController.php

```diff
@@ -181,13 +179,43 @@ public function postReassignuser(ReassignUser $request)
     }
 
     /**
-     * @param $user_id
+     * @param \Seat\Web\Http\Validation\NewIntelNote $request
+     * @param int                                    $user_id
      *
      * @return mixed
      */
-    public function editUserAccountStatus($user_id)
+    public function editUserAccountStatus(NewIntelNote $request, int $user_id)
     {
 
+        $user = $this->getUser($user_id);
+        $current_group = $user->group;
+
+        // Set default note
+        $note = collect(sprintf('This character has been reactivated by %s.',
+            auth()->user()->name));
+
+        if ($user->active){
+
+            // Initial setting of deactivation note
+            $note = collect(sprintf('This character has been deactivated by %s.', auth()->user()->name));
+
+            if ($current_group->users->count() > 1) {
+
+                // If deactivated user is part of a larger user group, use another note
+                $note = collect(sprintf('This character has been deactivated by %s and it formerly belonged to the following user group: %s.',
```

Comment:
Similarly here. Recording the group information will give a standard user more information that I believe they should know. Use the security log feature we already have for that.

Reply:
i moved the notes to the sec.log and removed the notes.